### ISSUE TITLE
dropdown list won't show overlay

### ISSUE THREAD

**Issue by SimonDarksideJ** [Unity-UI-Extensions/com.unity.uiextensions#51]
<sup>Issue created by Aswin Thomas as [Bitbucket Issue #&#8203;15](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/15) on 2015.04.23 18:35.</sup>
I created one dropdownlist and it works well. However, anymore lists created in the GUI won't work. The overlay seems to turn on/off, but it won't display. Any idea whats causing this?

**Comment by SimonDarksideJ** [comment]
<sup>On [2015.05.13 21:11](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/15#comment-34323289), michael hunt commented:</sup>
same here, any obvious reason/solution?

**Comment by SimonDarksideJ** [comment]
<sup>On [2015.06.13 20:07](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/15#comment-34323288), Jeff Smith commented:</sup>
The local scale of the Overlay and ScrollPanel gets reset to (0,0,0) for some reason. This also happens in ComboBox.cs. A fix for both classes is to add the two lines below: 


```c#

public void ToggleDropdownPanel(bool directClick)
{
    _overlayRT.transform.localScale = new Vector3(1, 1, 1);  //added by Jeff
    _scrollPanelRT.transform.localScale = new Vector3(1, 1, 1);  //added by Jeff 

```

Don't forget to set the ItemsToDIsplay=3 (or some non-zero number) too. Hope that helps someone.

**Comment by SimonDarksideJ** [comment]
<sup>On [2015.06.14 10:29](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/15#comment-34323287), michael hunt commented:</sup>
Thanks, really appreciated as my c# skills have a long way to go and i couldn't see the problem myself for the life
of me/

**Comment by SimonDarksideJ** [comment]
<sup>On [2015.06.14 11:25](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/15#comment-34323286), Jeff Smith commented:</sup>
Glad it helped!

**Comment by SimonDarksideJ** [comment]
<sup>On [2015.08.26 10:09](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/15#comment-34323285), @SimonDarksideJ commented:</sup>
reset local scale for overlay and scrollbar on toggle (fixes issue #15)

Fixed by @jeffssmith1 https://bitbucket.org/ddreaper/unity-ui-extensions/issues/15#comment-18792458

Referenced by 6fe42de5badd

**Comment by SimonDarksideJ** [comment]
<sup>On 2015.08.26 10:09 @SimonDarksideJ modified issue:</sup>
**status** changed `new` → `resolved`
